libs/s25main/figures/nofWorkman.cpp

```diff
@@ -110,3 +110,13 @@ MapPoint nofWorkman::FindPointWithResource(ResourceType type) const
 
     return MapPoint::Invalid();
 }
+
+std::vector<MapPoint> nofWorkman::FindAllPointsWithResource(ResourceType type) const
```

Comment:
See above: This is incredibly miner specific especially due to the `MINER_RADIUS` used below. So better put this into the miner class or even the function directly.